### PR TITLE
Uruguay: update holiday names

### PR DESCRIPTION
**Pull request by KJhellico**

### CODE REVIEW

File: holidays/countries/uruguay.py [hunk 1]
```diff
@@ -36,60 +35,64 @@ def _populate(self, year):
...
-        self[easter(year)] = "Día de Pascuas [Easter Day]"
+        self[easter_date] = "Día de Pascuas [Easter Day]"
 
         holiday_pairs = (
             (
                 # Desembarco de los 33 Orientales en la playa de la Agraciada.
                 date(year, APR, 19),
-                "Desembarco de los 33 Orientales Landing of the 33 Orientals "
-                "Aterrissagem dos 33 Orientais Sbarco dei 33 orientali",
+                "Desembarco de los 33 Orientales "
```
Comment (arkid15r) [inline holidays/countries/uruguay.py:89]: Could this fit a single line?

**Review by arkid15r — COMMENTED** [review]
Looks good!

File: holidays/countries/uruguay.py [hunk 2]
```diff
@@ -36,60 +35,64 @@ def _populate(self, year):
...
 
         # Partially paid holidays:
 
-        # Día de Reyes - Feriado en el cual se conmemora la llegada de
-        # los reyes magos a Jesus.
-        self[date(year, JAN, 6)] = "Día de Reyes"
+        # Día de los Niños.
+        self[date(year, JAN, 6)] = "Día de los Niños [Children's Day]"
 
         # Natalicio de José Gervacio Artigas.
```
Comment (arkid15r) [inline holidays/countries/uruguay.py:60]: I believe it should be `Gervasio` instead of `Gervacio` here and below.

Comment (KJhellico) [reply holidays/countries/uruguay.py:60]: Yes, exactly. Thank you!

File: holidays/countries/uruguay.py (hunk 1 above)
Comment (KJhellico) [reply holidays/countries/uruguay.py:89]: Yes, exactly fit :)

**Comment by dr-prodigy** [comment]
👍

**Pull request merged**